resources/logging/values.yaml

```diff
@@ -1,82 +1,146 @@
-    # The number of hours to retain the internal log collection service log files.
-    retention: "168h"
-    # The number of copies of each segment to replicate for log replication across the cluster for the internal log collection service log files. Should be at least 2.
-    replicationFactor: 1
-    # The number of hours before purging deleted segment files for the internal log collection service.
-    purge: "24h"
-    # The number of bytes to try and keep store segments for the internal log collection service.
-    targetSize: "134217728"
-  resources:
-    requests:
-      cpu: 125m
-      memory: 96Mi
-    limits:
-      memory: 1Gi
-
-# Local installation config
-isLocalEnv:
-  resources:
-    requests:
-      cpu: 125m
-      memory: 128Mi
-    limits:
-      memory: 512Mi
-  persistence:
-    size: 5Gi
-
-# Enable to add load balanced service.
-# Example annotation will create AWS internal LoadBalancer.
-# Ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
-loadBalancerService:
-  enabled: false
-  annotations:
-    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
-
-
-## Persist data to a persistent volume
-persistence:
-  enabled: true
-  accessMode: ReadWriteOnce
-  size: 200Gi
-  mountPath: /data
-
-nodeSelector: {}
-tolerations: []
-affinity: {}
+    auth_enabled: false
+    ingester:
+      lifecycler:
+        ring:
+          store: inmemory
+          replication_factor: 1
+    schema_configs:
+      - from: 0
+        store: boltdb
+        object_store: filesystem
+        schema: v9
+        index:
+          prefix: index_
+          period: 168h
+    storage_configs:
+      - name: boltdb
+        directory: /tmp/loki/index
+      - name: filesystem
+        directory: /tmp/loki/chunks
+
+promtail:
+  nameOverride: promtail
+  replicas: 1
+  deploymentStrategy: RollingUpdate
+
+  image:
+    repository: grafana/promtail
+    tag: master 
+    pullPolicy: Always
+
+  service:
+    port: 3100
+    annotations: {}
+    labels: {}
+
+  readinessProbe: {}
+
+  livenessProbe: {}
+
+  resources: {}
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  ## Pod Annotations
+  #podAnnotations:
+
+  ## Deployment annotations
+  # annotations: {}
+
+  ## Assign a PriorityClassName to pods if set
+  # priorityClassName:
+
+  ## Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  #
+  nodeSelector: {}
+
+  ## Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations:
+    - key: node-role.kubernetes.io/master
+      effect: NoSchedule
+
+
+  ## Affinity for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
 
 global:
   containerRegistry:
-    path: eu.gcr.io/kyma-project
+    path: docker.io/hbalik
```

Comment:
Still needs to be adjusted to the image resulting from the console integration

Reply:
image path updated